YubiKit/UnitTests/PIVPaddingTests.swift

```diff
@@ -174,8 +174,8 @@ struct PIVPaddingTests {
     @Test func unpadMalformedData() throws {
         let data = Data(
             hexEncodedString:
-                "00bcbb35b6ef5c94a85fb3439a6dabda617a08963cf81023bac19c619b024cb71b8aee25cc30991279c908198ba623fba88547741dbf17a6f2a737ec95542b56b2b429bea8bd3145af7c8f144dcf804b89d3f9de21d6d6dc852fc91c666b8582bf348e1388ac2f54651ae6a1f5355c8d96daf96c922a9f1a499d890412d09454"
-        )!
+                "00bcbb35b6ef5c94a85fb3439a6dabda617a08963cf81023bac19c619b024cb71b8aee25cc30991279c908198ba623fba88547741dbf17a6f2a737ec95542b56b2b429bea8bd3145af7c8f144dcf804b89d3f9de21d6d6dc852fc91c666b8582bf348e1388ac2f54651ae6a1f5355c8d96daf96c922a9f1a499d890412d09454AA"
```

Comment:
Looks like an extra byte was added to the end here. Was this intentional? What was the purpose?